rcl/test/rcl/test_events.cpp

```diff
@@ -134,11 +134,17 @@ class TestEventFixture : public ::testing::TestWithParam<TestIncompatibleQosEven
     // init publisher events
     publisher_event = rcl_get_zero_initialized_event();
     ret = rcl_publisher_event_init(&publisher_event, &publisher, pub_event_type);
+    if (ret == RCL_RET_UNSUPPORTED) {
+      GTEST_SKIP();
+    }
```

Comment:
i am not sure if this is the right thing to skip the test for specific rmw implementation though... that means if we introduce the bug that returns `RCL_RET_UNSUPPORTED` in rmw_fastrtps (just an example) accidently, this could not detect the bug at all. because the expectation is still `RCL_RET_OK`?

i would say using `rmw_get_implementation_identifier` with `rmw_zenoh_cpp` on top, then skip the test would be better thing here.

Reply:
> i would say using rmw_get_implementation_identifier with rmw_zenoh_cpp on top, then skip the test would be better thing here.

Sounds good to me!